### PR TITLE
Add org.synfig.SynfigStudio

### PR DESCRIPTION
**Pull request by eszlari**

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build org.synfig.SynfigStudio

**Comment by nedrichards** [comment]
Is there anyone from upstream who's interested in the package?
Can you send a PR upstream to add OARS and Version tags to the appdata, since it doesn't have it. https://github.com/synfig/synfig/blob/master/synfig-studio/synfigstudio.appdata.xml.in

**Comment by eszlari** [comment]
> Is there anyone from upstream who's interested in the package?

We will find out, after I have contacted upstream.

> Can you send a PR upstream to add OARS and Version tags to the appdata

Of course.

**Comment by nedrichards** [comment]
OK, well, build succeeded. https://flathub.org/builds/#/builders/2/builds/3215

The only blocker from my perspective is the "contacting people" and "upstream PR" ones.

File: org.synfig.SynfigStudio.yaml [hunk 1]
```diff
@@ -0,0 +1,211 @@
+app-id: org.synfig.SynfigStudio
+branch: stable
+runtime: org.gnome.Platform
+runtime-version: 3.28
```
Comment (TingPing) [inline org.synfig.SynfigStudio.yaml]: This is a string: `'3.28'`.

File: org.synfig.SynfigStudio.yaml [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    config-opts:
+      - --disable-static
+    cleanup:
+      - /share/doc
+    sources:
+      - type: archive
+        url: https://github.com/ImageMagick/ImageMagick6/archive/6.9.10-8.tar.gz
+        sha256: 82b1c536906228f5021be455e7b2bfbe6af189e0a3b257a7589632e645615815
+
+  - name: x264
```
Comment (TingPing) [inline org.synfig.SynfigStudio.yaml]: This is a string too: `'x264'

**Comment by eszlari** [comment]
> The only blocker from my perspective is the "contacting people" and "upstream PR" ones.

flatpak support issue: https://github.com/synfig/synfig/issues/550

PR for appdata: https://github.com/synfig/synfig/pull/598

**Comment by ice0** [comment]
Hi! Please also add @morevnaproject .

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.synfig.SynfigStudio

**Pull request closed without merging**